frontend/src/components/GoesData/AnimationStudioTab/StudioHistory.tsx

```
@@ -0,0 +1,61 @@
+                    <span className="px-2 py-1 text-xs bg-emerald-600/20 text-emerald-400 rounded">Done</span>
+                    {anim.output_path && (
+                      <a href={`/api/download?path=${encodeURIComponent(anim.output_path)}`}
+                        download className="text-xs text-primary hover:underline">Download</a>
+                    )}
+                  </>
+                )}
+                {anim.status === 'failed' && (
+                  <span className="px-2 py-1 text-xs bg-red-600/20 text-red-400 rounded" title={anim.error}>Failed</span>
+                )}
+                <button onClick={() => onDelete(anim.id)}
+                  className="p-1 text-gray-400 dark:text-slate-500 hover:text-red-400 transition-colors">
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)